cmd/neofs-cli/internal/common/exit.go

```diff
@@ -9,6 +9,10 @@ import (
 	"github.com/spf13/cobra"
 )
 
+// ErrAwaitTimeout is a predefined error variable that represents
```

Comment:
no need to clarify obvious fact that this is a predefined error varible
```suggestion
// ErrAwaitTimeout represents ...
```